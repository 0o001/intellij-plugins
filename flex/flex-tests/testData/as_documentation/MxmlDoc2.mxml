<?xml version="1.0" encoding="utf-8"?>

<mx:Application xmlns:mx="http://www.adobe.com/2006/mxml">
  <mx:DataGrid id="dataGrid" width="100%" height="100%" resizeEffect="aa">
    <mx:columns>
          <mx:DataGridColumn dataField="#ff<caret>ffff" />
    </mx:columns>
  </mx:DataGrid>
</mx:Application>