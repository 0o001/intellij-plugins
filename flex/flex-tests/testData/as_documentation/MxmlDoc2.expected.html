 AAA<p><b>Type:</b> <code>String</code><p>
<u>Current value:</u>&nbsp;<code>#ffffff</code><span style="background-color:#ffffff">&nbsp;&nbsp;&nbsp;</span></p>